independent-publisher-connectors/WebexIntegration/README.md

```diff
@@ -0,0 +1,32 @@
+# WebEx Custom Connector # 
+
+Team 18 from UCL Computer Science propose to build a custom connector for WebEx while working alongside the NHS.
+
+Our aim is to allow doctors to create meetings via PowerApp and send them via email or WhatsApp to the patients for remote diagnosis session, which would make the process simpler. 
+
+## Publisher: University College London ##
+
+## Prerequisites ##
+Cisco Webex Developer Account
```

Comment:
@ocarhui is it possible to provide a link for this page please? where to go to create an account?